pkg/types/validation/installconfig.go

```diff
@@ -555,3 +582,26 @@ func validateCloudCredentialsMode(mode types.CredentialsMode, fldPath *field.Pat
 	}
 	return allErrs
 }
+
+// validateURI checks if the given url is of the right format.
+func validateURI(uri string, fldPath *field.Path) (string, field.ErrorList) {
+	allErrs := field.ErrorList{}
+	parsed, err := url.ParseRequestURI(uri)
+	if err != nil {
+		return "", append(allErrs, field.Invalid(fldPath, uri, err.Error()))
+	}
+	if !parsed.IsAbs() {
```

Comment:
> Understanding that this is copied over from the cluster-network-operator, this check is not necessary. Since the URI is being parsed as a request URI, the URI must already be an absolute URI.



I'm not sure I follow. I thought this was just user input from the install config.

Reply:
I mean that the code here is copied from cluster-network-operator. The check of `IsAbs` comes from there. If we want to stay as a pristine copy of that code, we can keep the `IsAbs` check, but that check is not necessary.